Library/Homebrew/cmd/search.rb

```diff
@@ -89,15 +94,28 @@ def search
       remote_casks = remote_results[:casks]
       all_casks = local_casks + remote_casks
 
-      if all_formulae.any?
-        ohai "Formulae"
-        puts Formatter.columns(all_formulae)
-      end
+      if args.formulae?
```

Comment:
Similarly: what if `--formulae` and `--casks` are passed here? Probably want to output them both.

Reply:
Yes, makes sense!